src/Validate.js

```diff
@@ -0,0 +1,39 @@
+import { ERROR_MESSAGE, MAX_CAR_LENGTH } from "./Const.js";
+
+class Validate {
+  #throwIfInValid(condition, errorMessage) {
+    if (condition) {
+      throw new Error(errorMessage);
+    }
+    return this;
+  }
+
+  isLiminLength(name) {
+    return this.#throwIfInValid(
+      name.length > MAX_CAR_LENGTH,
+      ERROR_MESSAGE.belowLimit
+    );
+  }
+
+  isPositiveLength(name) {
+    return this.#throwIfInValid(name.length <= 0, ERROR_MESSAGE.positiveLength);
+  }
```

Comment:
👆 에 대한 예시로 이렇게 사용이 가능하겠네요
```suggestion
  isPositiveLength(name) {
    return this.#throwIfInValid({
        condition: name.length <= 0, 
        errorMessage: ERROR_MESSAGE.positiveLength
     });
  }
```

Reply:
요거는 취향이 담긴 의견이라 그냥 이런 관점도 있구나 정도로 참고해주세용 ㅎㅎ